src/iv/ivutils.h

```diff
@@ -10,7 +10,7 @@ OIIO_NAMESPACE_BEGIN
 /// representation.  Once optimized and tested, move to fmath.h
 
 inline float
-pow2roundupf(float f)
+ceil2f(float f)
 {
     float logval = logf(f) / logf(2.0f);
```

Comment:
Totally unrelated to this review, but this badly calls for a native implementation that just uses the float exponent :)